pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -66,21 +64,6 @@ func Register(ctx context.Context, management *config.ManagementContext) {
 		KontainerDriverLister: management.Management.KontainerDrivers("").Controller().Lister(),
 	}
 
-	local := &rkedialerfactory.RKEDialerFactory{
-		Factory: management.Dialer,
-		Ctx:     ctx,
-	}
-	docker := &rkedialerfactory.RKEDialerFactory{
-		Factory: management.Dialer,
-		Docker:  true,
-		Ctx:     ctx,
-	}
-	driver := service.Drivers[service.RancherKubernetesEngineDriverName]
-	rkeDriver := driver.(*rke.Driver)
-	rkeDriver.DockerDialer = docker.Build
-	rkeDriver.LocalDialer = local.Build
-	rkeDriver.WrapTransportFactory = docker.WrapTransport
-
 	c.backupClient.AddLifecycle(ctx, "etcdbackup-controller", c)
```

Comment:
We don't need this controller running at all and can be removed completely, are we planning to remove it in a separate PR?